docs/UserGuide.md

```diff
@@ -10,16 +10,19 @@ _Find Your Success with JobFindr_
 ## Table of Contents
 
 1. [Introduction](#introduction)
+    1. [First Time at JobFindr](#first-time-at-jobfindr)
```

Comment:
Could use another way to show listing instead of numbering each point again as it could be confusing

Reply:
Maybe I will try the {ToC} markdown in the next pr lets see if that's better